package.json

```diff
@@ -1,49 +1,54 @@
-{
-  "name": "@nodesecure/npm-security-fetcher",
-  "version": "1.0.0",
-  "description": "Fetch packages from npm for Security purposes",
-  "main": "index.js",
-  "type": "module",
-  "bin": {
-    "nsf": "./bin/index.js"
-  },
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/NodeSecure/npm-security-fetcher.git"
-  },
-  "keywords": [
-    "npm",
-    "security",
-    "fetcher",
-    "analyser"
-  ],
-  "author": "GENTILHOMME Thomas <gentilhomme.thomas@gmail.com>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/NodeSecure/npm-security-fetcher/issues"
-  },
-  "homepage": "https://github.com/NodeSecure/npm-security-fetcher#readme",
-  "devDependencies": {
-    "@nodesecure/eslint-config": "^1.3.1"
-  },
-  "dependencies": {
-    "@nodesecure/fs-walk": "^1.0.0",
-    "@nodesecure/js-x-ray": "^4.2.1",
-    "@nodesecure/npm-registry-sdk": "^1.3.0",
-    "@slimio/is": "^1.5.1",
-    "@slimio/lock": "^1.0.0",
-    "dotenv": "^16.0.0",
-    "filenamify": "^5.1.0",
-    "is-minified-code": "^2.0.0",
-    "kleur": "^4.1.4",
-    "klona": "^2.0.5",
-    "make-promises-safe": "^5.1.0",
-    "ms": "^2.1.3",
-    "pacote": "^13.0.5",
-    "qoa": "^0.2.0",
-    "sade": "^1.8.1"
-  }
-}
+{
+  "name": "@nodesecure/npm-security-fetcher",
+  "version": "1.0.0",
+  "description": "Fetch packages from npm for Security purposes",
+  "main": ".dist/index.js",
```

Comment:
You need to replace `main` with `exports` and `types`. Look at the following package.json:
https://github.com/NodeSecure/rc/blob/e0a6f3a3c0b8724c80bfaac3eab7e68c0da39cdc/package.json#L5-L7